gomockmatchersproto/proto_test_message.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+option go_package = "./gomockmatchers";
+
+package gomockmatchers;
+
+message TestRecord {
```

Comment:
Not sure if there is any other way to make this testable than adding an actual proto message here?

Reply:
This is a bit annoying and cluttering.

Can we create a mock of the `proto.Message` using gomock instead?

Reply:
I made use of the [structpb](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb) instead! (wasnt really sure how to go about mocking `proto.Message` tbh)